### DISCUSSION TITLE
19.02.1 crashes on iPadOS 17.3 when opening and playing any video for a few seconds

### DISCUSSION THREAD

**Discussion by HouseYeung** [qnblackcat/uYouPlus#1295] [Q&A]
I have installed the software, and it opens successfully, allowing me to browse recommended pages and settings. However, whenever I click on any video, the software crashes approximately a second later. Occasionally, some videos play normally as long as I don’t interact with the video playback window. How can this issue be resolved?

**Comment by qnblackcat** [comment]
Hm. Have you tried to disable uYou Playback controls in uYou’s setting?

**Reply by HouseYeung** [reply]
just disable the setting and it works， thank you

**Reply by jmorto11** [reply]
I'm having the same issue. I've been disabling anything uYou related I can see, but I must be missing something. Can you explain what needs to be disabled?

**Reply by jmorto11** [reply]
I was wrong. I had to disable isponsorblock. Hoping to find a workaround so I can get that back as it's one of my favorite features.
